src/Remotes/JlrVehicleRemoteInformation.ts

```diff
@@ -1,7 +1,7 @@
 import { QueryVehicleInformationService } from '../Services/QueryVehicleInformationService'
-import { VehicleRemoteAuthenticator } from './VehicleRemoteAuthenticator'
+import { VehicleRemoteAuthenticator, VehicleRemoteInformation } from './Types'
 
-class VehicleRemote {
```

Comment:
@ianCoops you'll notice that following this new structure, your original `VehicleRemote`, which was quite generic, has become the much more specific `JlrVehicleRemoteInformation`. This has also been changed to implement the generic `VehicleRemoteInformation` type,